src/setup-dev-server.js

```diff
@@ -79,10 +65,7 @@ module.exports = function setupDevServer(app, config, cb) {
         if (json.errors.length) {
             return;
         }
-        clientManifest = JSON.parse(readFile(
-            devMiddleware.fileSystem,
```

Comment:
The API changed so you can no longer access this filesystem via the returned `devMiddleware`.  So we now pass `outputFileSystem: mfs` and directly refer to `mfs` to read the updates files